appengine/standard/firebase/firetactoe/README.md

```diff
@@ -15,7 +15,7 @@ installed. You'll need this to test and deploy your App Engine app.
   console](https://firebase.google.com/console)
 * In the Overview section, click 'Add Firebase to your web app' and replace the
   contents of the file
-  [`templates/_firebase_config.html`](templates/_firebase_config.html) with the
+  [`templates/_firebase_config.json`](templates/_firebase_config.json) with the
```

Comment:
I kept this as html for simplicity reasons. This way, they can just copy/paste the code snippet from the firebase console into the template. If you change it to json, then they have to individually select each field, find the corresponding field in the json template, and paste each one. This leaves more room for error, and requires them to ignore the instructions and convenient "Copy" button in the firebase console.

I agree in principle that reading from json is clearer for someone reading the code, but I think there's a trade-off here between ease of setup and code readability. In this case, I figured the friction of figuring out what the regex line in the `get_firebase_db_url` did could be mitigated with a clear function name, docstring, and comments. (and was less than the alternative)